app-dicts/mecab-unidic/mecab-unidic-2.1.2.ebuild

```diff
@@ -12,7 +12,6 @@ SRC_URI="mirror://sourceforge.jp/${PN#*-}/58338/${MY_P}.zip"
 LICENSE="|| ( BSD GPL-2 LGPL-2.1 )"
 SLOT="0"
 KEYWORDS="~amd64 ~riscv"
```

Comment:
unzip at least is a BDEPEND

Reply:
I'll add that and make sure it isn't an REDEPEND too.